components/ui/CheckBox.tsx

```diff
@@ -24,10 +26,23 @@ export default function CheckBox({
 
   return (
     <TouchableOpacity
-      style={[styles.container, isChecked && styles.checkedContainer, style]}
+      style={[
+        styles.container,
+        isChecked && !unshaded && styles.checkedContainer,
+        unshaded && styles.unshadedContainer,
+        style,
+      ]}
       onPress={handleCheckedChange}
       {...rest}>
-      {isChecked ? <CheckIcon /> : <PlusIcon />}
+      {isChecked ? (
+        <CheckIcon
+          color={unshaded ? theme.colors.black : theme.colors.white}
+          height={unshaded ? 50 : 32}
+          width={unshaded ? 50 : 32}
```

Comment:
why is the height and width `50`, on the Figma it is `24*24`

Reply:
i thought 24 was small but let's keep it